src/pages/components/sheet.mdx

```diff
@@ -0,0 +1,199 @@
+import SheetSizeCode from '!!raw-loader!../../examples/chrome/SheetSize.tsx';
+
+<CodeExample code={SheetSizeCode}>
+  <SheetSize />
+</CodeExample>
+
+## API
+
+The Sheet component follows this structure:
+
+<!-- prettier-ignore -->
+```jsx
+<Sheet>
+  <Sheet.Header>
+    <Sheet.Title>
+      {/* a text to label the sheet */}
+    </Sheet.Title>
+    <Sheet.Description>
+      {/* a text describing the sheet */}
+    </Sheet.Description>
+  </Sheet.Header>
+  
+  <Sheet.Body>
+    {/* main content of the Sheet */}
+  </Sheet.Body>
+
+  <Sheet.Footer>
+    <Sheet.FooterItem>
+      {/* items to set in the footer, such as buttons */}
+    </Sheet.FooterItem>
+  </Sheet.Footer>
+
+  <Sheet.Close />
+</Sheet>
+```
+
+### Sheet
+
+<Component components={props.data.mdx.components} componentName="sheet" />
+
+The Sheet component provides state and accessibility attributes to its children.
+
+<PropSheet components={props.data.mdx.components} componentName="sheet" />
+
+### Sheet.Body
+
+The Sheet Body component is placed in the [`Sheet`](#sheet) component.
+This component requires accessible functionality to make the
+element focusable, if keyboard scrolling is needed.
+
+<Component components={props.data.mdx.components} componentName="sheet.body" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.body" />
+
+### Sheet.Title
+
+The Sheet Title labels the Sheet for assistive technology.
+This component is nested in the [`Sheet.Header`](#sheetheader) component.
+
+<Component components={props.data.mdx.components} componentName="sheet.title" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.title" />
+
+### Sheet.Description
+
+The Sheet Description describes the Sheet for assistive technology.
+This component is nested in the [`Sheet.Header`](#sheetheader) component.
+
+<Component components={props.data.mdx.components} componentName="sheet.description" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.description" />
+
+### Sheet.Header
+
+The Sheet Header component is placed in the [`Sheet`](#sheet) component.
+
+<Component components={props.data.mdx.components} componentName="sheet.header" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.header" />
+
+### Sheet.Footer
+
+The Sheet Footer component is placed in the [`Sheet`](#sheet) component.
+The component can alternate to compact styling.
+
+<Component components={props.data.mdx.components} componentName="sheet.footer" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.footer" />
+
+### Sheet.FooterItem
+
+The Sheet FooterItem component is nested in the [`Sheet.Footer`](#sheetfooter) component.
+
+<Component components={props.data.mdx.components} componentName="sheet.footeritem" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.footeritem" />
+
+### Sheet.Close
+
+The Sheet Close button component is placed in the [`Sheet`](#sheet) component.
```

Comment:
Is it worth identifying how it should be last for consistent tab order?

Reply:
Good call on this.